docs/usage-notes.md

```diff
@@ -22,6 +23,29 @@ Once the app is launched (either web or desktop version), it will prompt you to
 
 ![session_select](https://imgur.com/nBOy7zH.png)
 
+## Format of `profiles.json` 
```

Comment:
Since this feature is mainly for customisation, let's move this section out of the `Get Started` section.
We can create a new top-level and place it after `Notes on using the Desktop App`.

Also, let's rename the section to `Customising the available sessions`, as it's better to explain the feature from the user's perspective.